src/main.ts

```diff
@@ -671,6 +673,20 @@ function makeNiceGrpcServerStreamingMethodResult() {
   return { NiceGrpcServerStreamingMethodResult };
 }
 
+function makeGrpcWebErrorInterface() {
+  const GrpcWebError = conditionalOutput(
+    'GrpcWebError',
+    code`
+      export interface GrpcWebError extends Error {
```

Comment:
@zakhenry sorry for the late review here, this looks great!

Just curious, but wdyt about making this a class, like:

```
export class GrpcWebError extends Error {
  constructor(message: string, public code: grpc.Code, public metadata: grpc.Metadata) {
    super(message);
  }
}
```

As this would let callers do things like `if error instanceof GrpcWebError`.

Reply:
yep I like it, commit pushed to implement this idea.